main.py

```diff
@@ -81,78 +84,78 @@ def transform(census_data):
     census_data["tot_poc"] = census_data["tot_pop"] - census_data["tot_whitenh"]
     census_data["total_overcrowded_households"] = (
-        census_data["B25014_005E"]
-        + census_data["B25014_006E"]
-        + census_data["B25014_007E"]
-        + census_data["B25014_011E"]
-        + census_data["B25014_012E"]
-        + census_data["B25014_013E"]
+            census_data["B25014_005E"]
+            + census_data["B25014_006E"]
+            + census_data["B25014_007E"]
+            + census_data["B25014_011E"]
+            + census_data["B25014_012E"]
+            + census_data["B25014_013E"]
     )
     census_data["tot_home_lang"] = (
-        census_data["C16002_001E"] - census_data["C16002_002E"]
+            census_data["C16002_001E"] - census_data["C16002_002E"]
     )
     census_data["tot_wfh"] = census_data["B08101_001E"] - census_data["B08101_049E"]
 
     census_data["tot_unenrolled_school"] = census_data["B14001_001E"] - (
-        census_data["B14001_003E"]
-        + census_data["B14001_004E"]
-        + census_data["B14001_005E"]
-        + census_data["B14001_006E"]
-        + census_data["B14001_007E"]
+            census_data["B14001_003E"]
+            + census_data["B14001_004E"]
+            + census_data["B14001_005E"]
+            + census_data["B14001_006E"]
+            + census_data["B14001_007E"]
     )
     census_data["tot_wo_broadband"] = (
-        census_data["B28002_003E"] + census_data["B28002_013E"]
+            census_data["B28002_003E"] + census_data["B28002_013E"]
     )
     census_data["tot_less_vehicle"] = (
-        census_data["B08203_014E"]
-        + census_data["B08203_020E"]
-        + census_data["B08203_021E"]
-        + census_data["B08203_026E"]
-        + census_data["B08203_027E"]
-        + census_data["B08203_028E"]
+            census_data["B08203_014E"]
+            + census_data["B08203_020E"]
+            + census_data["B08203_021E"]
+            + census_data["B08203_026E"]
+            + census_data["B08203_027E"]
+            + census_data["B08203_028E"]
     )
     census_data["tot_mortgage_over35"] = (
-        census_data["B25091_009E"]
-        + census_data["B25091_010E"]
-        + census_data["B25091_011E"]
+            census_data["B25091_009E"]
+            + census_data["B25091_010E"]
+            + census_data["B25091_011E"]
     )
     census_data["tot_rent_over35"] = (
-        census_data["B25070_008E"]
-        + census_data["B25070_009E"]
-        + census_data["B25070_010E"]
+            census_data["B25070_008E"]
+            + census_data["B25070_009E"]
+            + census_data["B25070_010E"]
     )
     census_data["tot_eng_prof"] = (
-        census_data["C16002_004E"]
-        + census_data["C16002_007E"]
-        + census_data["C16002_010E"]
-        + census_data["C16002_013E"]
+            census_data["C16002_004E"]
+            + census_data["C16002_007E"]
+            + census_data["C16002_010E"]
+            + census_data["C16002_013E"]
```

Comment:
I'm just picking these lines at semi-random, but do you mind running this through `black` to lint it? I just did it to test out the suggestion here, and I think it does a good job cleaning up this code and keeping it readable.